src/gui/optionsdialog.ui

```diff
@@ -130,7 +130,7 @@
             <item>
              <widget class="QGroupBox" name="UISettingsBox">
               <property name="title">
-               <string>Interface</string>
+               <string>Interface (Requires Restart)</string>
```

Comment:
IMO it's not intuitive to understand that this applies to all options in the groupbox.

Reply:
I find it redundant to add this with every option. Any suggestions?

Reply:
>I find it redundant to add this with every option.

I don't see why... it's not like the options are intertwined, at least language and theme. Anyway just my opinion. I'm not some GUI design expert.

Reply:
If possible I would like to hear the opinions of @chcobo1 and @glassez before making furthur changes.